azure-pipelines.yml

```diff
@@ -1,36 +1,30 @@
+
 trigger:
   batch: true
   branches:
     include:
       - "*"
 
 variables:
-- name: SolutionBaseName
-  value: __SolutionBaseName__
-- name: BuildConfiguration
-  value: release
 - name: BuildPlatform
-  value: any cpu
+  value : 'any cpu'
+- name: BuildConfiguration
+  value: 'release'
 
 pool:
-  name: DAS - Continuous Integration
   workspace:
     clean: all
+  name: 'DAS - Continuous Integration'
 
 resources:
   repositories:
-  - repository: self
```

Comment:
If the self reference isn't needed, good if you can PR its removal from  the das-github-template https://github.com/SkillsFundingAgency/das-github-template/blob/main/azure-pipelines.yml#L22